CHANGELOG.md

```diff
@@ -5,6 +5,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## [Unreleased]
 
+## [2.1.0] - 2018-12-06
+### Added
+- Use wait_until_available to reduce task runs ([BOLT-956](https://tickets.puppetlabs.com/browse/BOLT-956))
```

Comment:
This ticket number was accidental, it no longer exists :(

Reply:
956 exists.  The commit says 957

Reply:
Oh ok. Close enough.